UPGRADING.md

```diff
@@ -40,29 +41,29 @@ With an entry in your `plist` file:
 </dict>
 ```
 
-You can add further configuration options to the `plist` or construct a `BugsnagConfiguration` to set further options in code. For full details, see the online docs
+You can add further configuration options to your `Info.plist` or construct a `BugsnagConfiguration` to set further options in code. For full details, see the online docs
 ([iOS](https://docs.bugsnag.com/platforms/ios/configuration-options/#setting-configuration-options) |
  [macOS](https://docs.bugsnag.com/platforms/macos/configuration-options/#setting-configuration-options) |
- [tvOS](https://docs.bugsnag.com/platforms/tvos/configuration-options/#setting-configuration-options)).
+ [tvOS](https://docs.bugsnag.com/platforms/tvos/configuration-options/#setting-configuration-options).)
 
 #### Additions
```

Comment:
If `notifyError:withData` wasn't present in the last release, `notifyError:block` should probably be added to this section.

Reply:
`notifyError:withData` wasn't present but `notifyError:block` was, so I don't believe there are any changes for anyone using `notifyError` in v6.